### PR TITLE
pull before pushing

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@mariusvniekerk what is the best way to resolve conflicts?

**Comment by scopatz** [comment]
You can use the `thiers` merge strategy

File: conda_forge_tick/cli.xsh [hunk 1]
```diff
@@ -15,7 +15,8 @@ def deploy(args):
     if args.dry_run:
         print("(dry run) deploying")
         return
-    for cmd in [['git', 'add', 'pr_json/*'],
+    for cmd in ['git', 'pull', '--ff-only'],
```
Comment (jakirkham) [inline conda_forge_tick/cli.xsh]:
```suggestion
    for cmd in [['git', 'pull', '--ff-only'],
```

File: conda_forge_tick/cli.xsh [hunk 2]
```diff
@@ -15,7 +15,8 @@ def deploy(args):
     if args.dry_run:
         print("(dry run) deploying")
         return
-    for cmd in [['git', 'add', 'pr_json/*'],
+    for cmd in [['git', 'pull', '--ff-only'],
```
Comment (CJ-Wright) [inline conda_forge_tick/cli.xsh]:
```suggestion
    for cmd in [['git', 'pull', '-s', 'recursive', '-X', 'theirs'],
```

**Pull request merged**